package.json

```diff
@@ -41,10 +46,15 @@
     "react-select": "^5.1.0",
     "react-simple-code-editor": "^0.11.2",
     "react-table": "^7.7.0",
-    "react-tooltip": "^4.2.21"
+    "react-tooltip": "^4.2.21",
+    "react-usestateref": "^1.0.8"
```

Comment:
why do we need this one, dont think its coming from ethereumjs?

Reply:
Hey, you're right, this was just some leftovers I forgot to clean from when I was testing some stuff.
Removing this package.